nativescript/src/tns-java-classes.ts

```diff
@@ -0,0 +1,25 @@
+//Resolve JavaScript classes that extend a Java class, and need to resolve
+//their JavaScript module from a bundled script. For example:
+//NativeScriptApplication, NativeScriptActivity, etc.
+//
+//This module gets bundled together with the rest of the app code and the
+//`require` calls get resolved to the correct bundling import call.
+//
+//At runtime the module gets loaded *before* the rest of the app code, so code
+//placed here needs to be careful about its dependencies.
```

Comment:
This file should be removed, it's from the old-version of `nativescript-dev-webpack`